### PR TITLE
UI - strata hotel application

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-274-nevq2lox.web.app

File: strr-base-web/app/components/connect/form/field/Index.vue [hunk 1]
```diff
@@ -18,7 +18,9 @@ defineProps({
 })
 
 const normalize = () => {
-  model.value = normalizeInput(model.value)
+  if (typeof model.value === 'string') {
+    model.value = normalizeInput(model.value)
```
Comment (deetz99) [inline strr-base-web/app/components/connect/form/field/Index.vue:22]: Just wondering if using the .trim accomplishes the same thing? The only thing it doesn't do is replace undefined with an empty string

https://vuejs.org/guide/essentials/forms.html#trim

**Review by deetz99 — APPROVED** [review]
Everything looks pretty good to me 👍 

We don't have the ci/cd for strata yet right?

File: strr-base-web/app/components/connect/form/field/Index.vue (hunk 1 above)
Comment (kialj876) [reply strr-base-web/app/components/connect/form/field/Index.vue:22]: oooh nice yeah I'll try that

**Review by thorwolpert — APPROVED** [review]

File: strr-strata-web/app/stores/strataApplication.ts [hunk 2]
```diff
@@ -27,7 +26,7 @@ export const useStrrStrataApplicationStore = defineStore('strr/strataApplication
 
   const validateStrataConfirmation = (returnBool = false): MultiFormValidationResult | boolean => {
     const result = validateSchemaAgainstState(
-      getConfirmationSchema(), platformConfirmation, 'platform-confirmation-form'
+      confirmationSchema, confirmation, 'platform-confirmation-form'
```
Comment (deetz99) [inline strr-strata-web/app/stores/strataApplication.ts:29]: Should we update this id to 'strata-confirmation-form' just to avoid confusion?

Comment (kialj876) [reply strr-strata-web/app/stores/strataApplication.ts:29]: yes definitely, I will update

**Review by deetz99 — APPROVED** [review]
👍

File: strr-strata-web/app/stores/strataApplication.ts (hunk 2 above)
Comment (deetz99) [reply strr-strata-web/app/stores/strataApplication.ts:29]: Cool, other than looks good! 🎉

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-274-nevq2lox.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-274-nevq2lox.web.app

**Pull request merged**